lang/cs/Org.Apache.REEF.Tang/Formats/AvroConfigurationDataContract/AvroConfiguration.cs

```diff
@@ -37,7 +38,7 @@ public AvroConfiguration()
         }
 
         [DataMember]
-        public HashSet<ConfigurationEntry> Bindings { get; set; }
+        public List<ConfigurationEntry> Bindings { get; set; }
```

Comment:
We are in transition. Much code uses the implementations still. It would be better to use the interface type here, `IList`.